MapboxNavigation/RouteMapViewController.swift

```diff
@@ -59,7 +59,25 @@ class RouteMapViewController: UIViewController {
     weak var routeController: RouteController!
     let distanceFormatter = DistanceFormatter(approximate: true)
     var arrowCurrentStep: RouteStep?
-    var isInOverviewMode = false
+    var isInOverviewMode = false {
+        didSet {
+            if isInOverviewMode {
+                overviewButton.isHidden = true
+                recenterButton.isHidden = false
+                wayNameView.isHidden = true
+                mapView.logoView.isHidden = true
+            } else {
+                overviewButton.isHidden = false
+                recenterButton.isHidden = true
+                mapView.logoView.isHidden = false
+            }
+            
+            if let controller = routePageViewController.currentManeuverPage {
+                controller.step = currentStep
+                routePageViewController.updateManeuverViewForStep()
```

Comment:
@1ec5 What's the purpose of updating the page here?
It looks like `controller.step = currentStep` will be overriden by `upComingStep ?? currentStep` in `updateManeuverViewForStep()`.